diff -aur sysbench-0.4.12.orig.orig/Makefile.am sysbench-0.4.12.orig/Makefile.am
--- sysbench-0.4.12.orig.orig/Makefile.am	2014-12-11 09:56:42.692412514 +0100
+++ sysbench-0.4.12.orig/Makefile.am	2014-12-11 10:11:32.036445517 +0100
@@ -14,6 +14,6 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
-SUBDIRS = doc sysbench
+SUBDIRS = sysbench
 
 EXTRA_DIST = autogen.sh README README-WIN.txt ChangeLog
